jdaviz/core/helpers.py

```diff
@@ -265,3 +267,106 @@ def get_model_parameters(self, models=None, model_label=None, x=None, y=None):
+        Display the jdaviz application in a "sidecar", which by default is a tab
+        on the right side of the jupyterlab  interface.
+
+        Parameters
+        ----------
+        title : str, optional
+            The title of the sidecar tab.  Defaults to the name of the
+            application - e.g. "specviz".
+        anchor : str
+            Where the tab should appear, by default on the right.  To see all
+            options see the ``sidecar.Sidecar.anchor.values``.
+
+        additional keywords are passed into the ``sidecar.Sidecar`` constructor.
+        See ``jupyterlab-sidecar`` for the most up-to-date options.
+
+        Returns
+        -------
+        sidecar
+            The ``sidecar.Sidecar`` object used to create the tab.
+
+        Notes
+        -----
+        The ``jupyterlab-sidecar`` module needs to be installed for this method
+        to work.  ``pip install sidecar`` should suffice for most use cases.
+
+        If this method is called in the "classic" Jupyter notebook, the app will
+        appear inline, as only lab has a mechanism to have multiple tabs.
+
+        See Also
+        --------
+        show_in_new_tab
+        show_inline
+        """
+        try:
+            from sidecar import Sidecar
+        except ModuleNotFoundError as e:
+            raise ModuleNotFoundError('You need to install jupyterlab-sidecar '
+                                      'to use `show_in_sidecar`.  Try `pip '
+                                      'install sidecar` and re-start your '
+                                      'Jupyter server.') from e
+
+        if 'title' not in kwargs:
+            kwargs['title'] = self.app.config
+
+        scar = Sidecar(**kwargs)
+        with scar:
+            display(self.app)
+
+        return scar
+
+    def show_in_new_tab(self, **kwargs):
+        """
+        Display the jdaviz application in a new tab in Jupyterlab.
+
+        Parameters
+        ----------
+        title : str, optional
+            The title of the sidecar tab.  Defaults to the name of the
+            application - e.g. "specviz".
+
+        Additional keywords are passed into the ``sidecar.Sidecar`` constructor.
+        See ``jupyterlab-sidecar`` for the most up-to-date options.
+
+        Returns
+        -------
+        sidecar
+            The ``sidecar.Sidecar`` object used to create the tab.
+
+        Notes
+        -----
+        The ``jupyterlab-sidecar`` module needs to be installed for this method
+        to work.  ``pip install sidecar`` should suffice for most use cases.
+
+        If this method is called in the "classic" Jupyter notebook, the app will
+        appear inline, as only lab has a mechanism to have multiple tabs.
+
+        See Also
+        --------
+        show_in_sidecar
+        show_inline
+        """
+        if 'anchor' in kwargs:
+            if 'tab' not in kwargs['anchor']:
+                raise ValueError('show_in_new_tab cannot have a non-tab anchor')
```

Comment:
Maybe it is nicer to fallback to default?
```suggestion
                warnings.warn('show_in_new_tab cannot have a non-tab anchor, using tab-after', UserWarning)
                kwargs['anchor'] = 'tab-after'
```